.github/workflows/pythonapp.yml

```diff
@@ -47,9 +47,9 @@ jobs:
     - name: Install pytorch 1.5.1+cpu if python 3.5 (not mac)
       if: ${{ matrix.python-version == '3.5' && matrix.os != 'macos-10.14' }}
       run: pip install torch==1.5.1+cpu -f https://download.pytorch.org/whl/torch_stable.html
-    - name: Install pytorch 1.7.0+cpu if python > 3.5 (not mac)
+    - name: Install pytorch nightly if python > 3.5 (not mac)
       if:  ${{ matrix.python-version != '3.5' && matrix.os != 'macos-10.14' }}
-      run: pip install torch==1.7.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
+      run: pip install --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
```

Comment:
I am not sure we want to run our cicd on their nightly built. Since it's not stable we may get intro troubles.

Reply:
ok, since we need 1.8 but it's not in pypi yet, do you suggest we pin it to a [specific version](https://download.pytorch.org/whl/nightly/cu110/torch_nightly.html) of 1.8? If so, which?

Reply:
We need 1.8 only for exporting to onnx, pytorch should still work (TVM, I don't know). So I think it is safer to leave 1.7 in the pipeline and move to 1.8 once it's stable. When the onnx export is used, we can either leave as it is (so it will say `nonzero` is not currently supported), or we can add a specific exception, like label encoder not supported until pytorch 1.8.

Reply:
k sounds good